python/cuml/test/dask/test_datasets.py

```diff
@@ -171,14 +170,14 @@ def test_make_regression(n_samples, n_features, n_informative,
                 assert coefs_part.flags['C_CONTIGUOUS']
 
 
-@pytest.mark.parametrize('n_samples', [500, 1000])
-@pytest.mark.parametrize('n_features', [50, 100])
+@pytest.mark.parametrize('n_samples', [unit_param(500), stress_param(1000)])
+@pytest.mark.parametrize('n_features', [unit_param(50), stress_param(100)])
 @pytest.mark.parametrize('hypercube', [True, False])
 @pytest.mark.parametrize('n_classes', [2, 4])
 @pytest.mark.parametrize('n_clusters_per_class', [2, 4])
 @pytest.mark.parametrize('n_informative', [7, 20])
 @pytest.mark.parametrize('random_state', [None, 1234])
-@pytest.mark.parametrize('n_parts', [2, 23])
+@pytest.mark.parametrize('n_parts', [unit_param(4), stress_param(23)])
```

Comment:
Any specific reason we went from 2 to 4 with this parameter?

Reply:
This could definitely be 2 again, 4 just felt like it might be more appropriate if its an only test?

Reply:
p.s., totally agree about further possible reductions